index.js

```diff
@@ -30,7 +33,10 @@ function sleep(time) {
 core.group('Execute Test', async () => {
     // 1. Trigger Test
     const envString = ENVIRONMENT ? `&enviroment=${ENVIRONMENT}` : '';
+    const instance = INSTANCE;
```

Comment:
I think if we make this the full domain w/ .com then we can just call this DOMAIN